frontend/javascripts/test/backend-snapshot-tests/datasets.e2e.ts

```diff
@@ -29,27 +37,28 @@ test.serial("getDatasets", async (t) => {
   writeTypeCheckingFile(datasets, "dataset", "APIDatasetCompact", {
     isArray: true,
   });
-  t.snapshot(datasets);
+  console.log(datasets);
```

Comment:
I’m not super familiar with the e2e tests. Do they typically console.log stuff, in addition to writing the snapshots? If not, maybe remove this line again?

Reply:
Yes, simply forgotten